string/strcspn.c

```diff
@@ -27,46 +27,65 @@
 
 /* Return the length of the maximum initial segment of S
    which contains no characters from REJECT.  */
-size_t
-STRCSPN (const char *str, const char *reject)
+size_t STRCSPN (const char *str, const char *reject)
 {
   if (__glibc_unlikely (reject[0] == '\0') ||
       __glibc_unlikely (reject[1] == '\0'))
     return __strchrnul (str, reject [0]) - str;
 
   /* Use multiple small memsets to enable inlining on most targets.  */
   unsigned char table[256];
-  unsigned char *p = memset (table, 0, 64);
```

Comment:
Would you prefer

```
unsgined char *p=
memset (table,0,64);
```
Also on a side note, I never touched this code since there was no way I could optimize this area.